components/card/card.css

```diff
@@ -0,0 +1,162 @@
+@import url(../utilities.css);
+@import url(../../quik.css);
+
+
+.card {
+	border-radius: 5px;
+    padding: 2rem;
+    display: flex;
+    border: 2px solid var(--secondary);
+    position: relative;
+}
+
+
+/* text only card */
+.card-text-only{
+    max-width: 30rem;
+    height: max-content;
+    overflow-y: auto;
```

Comment:
we already have defined max-content and flex-direction as column so it will adjust according to content

so overflow can be removed
also i would like to know your use case behind this

Reply:
if the text content is overflowing you can use ellipsis for that

means whenever the content will overflow it will add ... at the end of paragraph